Frontend/src/app/SuccessScreen.js

```diff
@@ -34,6 +36,17 @@ export default function SuccessScreen() {
       <TouchableOpacity style={{...GlobalStyles.button, width: '200px'}} onPress={handleDone}>
         <Text style={GlobalStyles.buttonText}>Done</Text>
       </TouchableOpacity>
+
+      {fileID && eventID && (
+        <View style={{ alignItems: 'center', marginTop: spacing.lg }}>
+          <View style={{ backgroundColor: 'white', padding: 20, borderRadius: 10 }}>
+            <QRCode
+              value={`https://a2odysseylabs.github.io/PhotoShare/#/aibooth/${eventID}/${fileID}`}
```

Comment:
For now I just created the QR code in the SucessScreen, but can you help me to create a new screen for all the email, text and QR inputs, so that users can choose it.